neuro_san/service/watcher/registries/registry_storage_updater.py

```diff
@@ -77,6 +79,15 @@ def start(self):
                          self.manifest_path, self.update_period_in_seconds)
         self.observer.start()
 
+    def log_next_update_time(self):
+        """
+        Log the time for next manifest file update check
+        """
+        # Log the NEXT update time
+        next_time = time.time() + self.get_update_period_in_seconds()
+        next_time_fmt: str = datetime.datetime.fromtimestamp(next_time).isoformat()
+        self.logger.info("Next time manifest will be checked for updates: %s", next_time_fmt)
+
```

Comment:
Self-explanatory: compute next time (approx, but good enough) when "update_storage" will be called,
and log it in standard ISO format.